board/safety/safety_volkswagen.h

```diff
@@ -38,6 +38,7 @@ addr_checks volkswagen_mqb_rx_checks = {volkswagen_mqb_addr_checks, VOLKSWAGEN_M
 #define MSG_MOTOR_3     0x380   // RX from ECU, for driver throttle input
 #define MSG_GRA_NEU     0x38A   // TX by OP, ACC control buttons for cancel/resume
 #define MSG_BREMSE_3    0x4A0   // RX from ABS, for wheel speeds
```

Comment:
Can garbage-collect the BREMSE_3 definition since we're no longer using it in the safety.